src/k8s/pkg/k8s/utils/file.go

```diff
@@ -11,8 +11,22 @@ import (
 	"syscall"
 
 	"github.com/canonical/k8s/pkg/snap"
+	"github.com/moby/sys/mountinfo"
 )
 
+// GetMountPath returns the first mountpath for a given filesystem type.
+func GetMountPath(fsType string) (string, error) {
+	mounts, err := mountinfo.GetMounts(mountinfo.FSTypeFilter(fsType))
+	if err != nil {
+		return "", fmt.Errorf("failed to find the mount info for %s: %w", fsType, err)
+	}
+	if len(mounts) == 0 {
+		return "", fmt.Errorf("failed to find the mount info for %s", fsType)
```

Comment:
`fmt.Error("could not find any %s filesystem mount", fsType)`

seems more accurate.